subcommand/inject-connect/command.go

```diff
@@ -145,8 +157,16 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagLifecycleSidecarMemoryLimit, "lifecycle-sidecar-memory-limit", "50Mi", "Lifecycle sidecar memory limit.")
 
 	c.http = &flags.HTTPFlags{}
+
 	flags.Merge(c.flagSet, c.http.Flags())
 	c.help = flags.Usage(help, c.flagSet)
+
+	// wait on an interrupt for exit, be sure to init it before running
+	// the controller so that we dont receive an interrupt before its ready
```

Comment:
```suggestion
	// Wait on an interrupt for exit, be sure to init it before running
	// the controller so that we don't receive an interrupt before it's ready.
```